src/graphcast_agent/mod.rs

```diff
@@ -456,9 +458,6 @@ impl GraphcastAgent {
             "Selected content topic from subscriptions"
         );
 
-        // Check network before sending a message
-        self.network_check()
```

Comment:
would you please briefly explain why this function is no longer called?

Reply:
this function is very slow now that we're using waku's discovery ENR, it takes about 250-300s each time and it blocks message sending, I opted into calling it every 5 minutes in [the Radio itself](https://github.com/graphops/subgraph-radio/pull/133/files#diff-55d1607ee64e5b063ec6a5655e6af764f63c12f6b7f78bc0509c73693efcc23dR406-R410)

Reply:
depending on whether network connectivity should be handled by the sdk, we can later on refactor the function to make sure it doesn't take that long, but okay here